main.py

```diff
@@ -1,6 +1,18 @@
 # KaanBOT is a turkish discord bot. It is programmed to work in a single server.
 
 # Importing packages.
+import os
+import requests
+
+try:
+  import requests
+except ImportError:
+  print("Trying to Install required module: requests\n")
+  os.system('python -m pip install requests')
```

Comment:
I think "pip install -r requirements.txt" command is better way to import packages.